src/common/FileSystem.cpp

```diff
@@ -2699,6 +2733,22 @@ const PakInfo* FindPak(Str::StringRef name, Str::StringRef version, uint32_t che
 	return &*(iter - 1);
 }
 
+const PakInfo* FindPak(Str::StringRef name, Str::StringRef version, uint32_t checksum)
```

Comment:
FindPak with a checksum doesn't need a prefix-aware implementation. This is for making the client load the exact same paks as the server so it should always be an exact match.

Reply:
OK, I removed the wrapper.